seedGenerator.js

```diff
@@ -0,0 +1,36 @@
+const generator = require('./database/helpers/generator.js');
+const csvWriter = require('csv-write-stream');
+const writer = csvWriter();
+const fs = require('fs');
+// const db = require('./postgresdb/postgres.js');
```

Comment:
In a similar case to the above; if you don’t need the commented out line, it’s recommended that it’s removed. On top of making it organized, it’ll also make your PR even more clean and amazing looking; not that it’s not amazing already

Reply:
This one I remember removing, so I'm assuming that it was part of the stuff I lost in my git issues.
Good catch :-)